storage/aliases.go

```diff
@@ -14,10 +14,7 @@
 
 package storage
 
-import (
-	"github.com/google/trillian/storage/storagepb"
-	"github.com/google/trillian/storage/tree"
```

Comment:
This builds without `tree` being imported?

Reply:
Look 2 lines below :)